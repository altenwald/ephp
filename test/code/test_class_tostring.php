<?php

class A {}
class B {
    public function __toString() {
        return "B<class>";
    }
}

$a = new A();
$b = new B();

print $b . "\n";
print $a . "\n";

