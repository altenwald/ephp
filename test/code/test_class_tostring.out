B<class>

Catchable fatal error: Object of class A could not be converted to string in {{CWD}}/test/code/test_class_tostring.php on line 14
